### ISSUE TITLE
cant rejoin created world

### ISSUE THREAD

**Issue by DivideByPotato** [Haggle1996/RevolutionPack#505]
I made a world and played it for a while, then a little while later i rejoined the world and it loaded for a while, and the put me back to the title screen. can someone please try to fix this problem?

**Comment by Aivech** [comment]
Have you tried waiting 5 minutes?

**Comment by jusvit** [comment]
Did you prefen it?

**Comment by jusvit** [comment]
Prefen*

**Comment by jusvit** [comment]
Pregen, stupid autocorrect

**Comment by Aivech** [comment]
I'm serious. Sometimes, if you wait a while, the world will load. 

Also, please provide the FML client latest and latest.txt log files. They're too big for Pastebin, use gist.github.com or similar.

**Comment by Haggle1996** [comment]
If the console window slowly reports new rivers being generated at exceptionally large coordinates (20,000 + or -) endlessly, you'll need to roll back to a backup.

**Comment by camoceltic** [comment]
I'm having the same problem.  Logs included in the gist below.  After hitting "Play Selected World" on the world I want to play on, it goes to the loading screen, gets to "Building Terrain" (may be wrong on the second word), then returns to the main menu.  After waiting 10 minutes, I tried to join the world again, but it goes straight to "Shutting down internal server" and freezes.

https://gist.github.com/camoceltic/974430e4690fdf31dcb25e917168bc35

EDIT: Changed "Building World" to "Building Terrain"
